python/dllib/examples/run-example-tests-yarn-tf1.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
move to python/orca/example

Reply:
Updated in this [commit](https://github.com/intel-analytics/BigDL/pull/3756/commits/ae101155b6bba211fbe6f67a2d90b70603a8d29b).